HtmlEditor: Add Keyboard Trap Fix Info (#9042)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxHtmlEditor/1 Configuration/customizeModules.md

> The DevExtreme Quill modules and the API you can use to customize them are described in the <a href="https://github.com/DevExpress/devextreme-quill/tree/master/docs/modules" target="_blank">Modules</a> documentation section. For example, the <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/history.md" target="_blank">History</a> module, which handles the undo and redo operations, can be customized as follows:

Reviewer: ```suggestion
The  <a href="https://github.com/DevExpress/devextreme-quill/tree/master/docs/modules" target="_blank">Modules</a> documentation section describes the DevExtreme Quill modules and the API you can use to customize them. For example, the <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/history.md" target="_blank">History</a> module, which handles the undo and redo operations, can be customized as follows:
```

    import { DxHtmlEditorModule } from 'devextreme-angular/ui/html-editor';

    // ...
    export class AppComponent {
        customizeQuillModules(config) {
            config.history = {
                delay: 0,
                maxStack: 5000,
            }; 
        }
    }

##### Vue

    <!-- tab: App.vue -->
    <template>
        <DxHtmlEditor
            :customize-modules="customizeQuillModules"
        />
    </template>

    <script setup lang="ts">
    import { DxHtmlEditor } from 'devextreme-vue/html-editor';

    function customizeQuillModules(config) {
        config.history = {
            delay: 0,
            maxStack: 5000,
        }; 
    }
    </script>

##### React

    <!-- tab: App.tsx -->
    import React, { useCallback } from 'react';
    import { HtmlEditor } from 'devextreme-react/html-editor';

    export default function App() {
        const customizeQuillModules = useCallback((config) => {
            config.history = {
                delay: 0,
                maxStack: 5000,
            };
        }, []);

        return (
            <HtmlEditor
                customizeModules={customizeQuillModules}
            />
        );
    }

---

> You can configure **customizeModules** to modify the [keyboard navigation](/Documentation/Guide/UI_Components/HtmlEditor/Accessibility/#Keyboard_Navigation) behavior of HTML Editor using `keyboard.inlineTabInsertion`:

Reviewer: ```suggestion
Use **customizeModules** and `keyboard.inlineTabInsertion` to modify the HTML Editor's [keyboard navigation](/Documentation/Guide/UI_Components/HtmlEditor/Accessibility/#Keyboard_Navigation) behavior:
```

---

##### jQuery

    <!-- tab: index.js -->
    $("#html-editor").dxHtmlEditor({
        customizeModules(config) {

>             config.keyboard.inlineTabInsertion = false;

Reviewer 2: config.keyboard.inlineTabInsertion = false - it is default HtmlEditor value. If user would change the default value, he would set it to true.
